docs/programming-guide.md

```diff
@@ -24,7 +24,8 @@ along with if you launch Spark's interactive shell -- either `bin/spark-shell` f
 
 <div data-lang="scala"  markdown="1">
 
-Spark {{site.SPARK_VERSION}} uses Scala {{site.SCALA_BINARY_VERSION}}. To write
+Spark {{site.SPARK_VERSION}} is built and distributed to work with Scala {{site.SCALA_BINARY_VERSION}} 
+by default. (Spark can be built to work with other versions of Scala, too.) To write
```

Comment:
The releases have to be built for one particular version, and in this case that's 2.10 right now. In Maven it's released for 2.10 and 2.11, yes. But you'd have to rebuild Spark for 2.11 to deploy Spark as a cluster. You would not have to rebuild it to compile against Spark + Scala 2.11.